index.html

```diff
@@ -186,11 +186,14 @@ <h2 id="constraints">Constraining Display Surface Selection</h2>
 
       <section>
         <h2>New Constraints for Captured Display Surfaces</h2>
-        <p>Two constraints are defined that allow an application to observe
+        <p>Three constraints are defined that allow an application to observe
         properties of the selected <a>display surface</a>.  Since the source of
         media cannot be changed after a <code>MediaStreamTrack</code> has been
         returned, these constraints cannot be changed by an application.</p>
```

Comment:
This description seems not accurate any more, since the new constraint modifies a property of the captured surface. Can you rephrase?